src/views/ProfileCreation/config.ts

```diff
@@ -3,6 +3,6 @@ import { parseUnits } from '@ethersproject/units'
 export const MINT_COST = parseUnits('1')
 export const REGISTER_COST = parseUnits('0.5')
 export const ALLOWANCE_MULTIPLIER = 5
-export const STARTER_BUNNY_IDENTIFIERS = ['sleepy', 'dollop', 'twinkle', 'churro', 'sunny']
+export const STARTER_NFT_BUNNY_IDS = ['5', '6', '7', '8', '9']
```

Comment:
Should use constants here to remember what those ids are?

Reply:
Yeah, agreed. Or at the very least we should comment what each bunny id is.